### PR TITLE
add some interaction and eslint

### PR DESCRIPTION
**Pull request by stephenkingsley**
1. add eslint rule
2. add contextmenu event. when click photos or videos, this app will show the cover display it.
3. fix drag. when scroll to next photo, this app can not move.
4. save photos and videos

### CODE REVIEW

**Comment by terkelg** [comment]
Hi @stephenkingsley. Thanks for contributing. There's some good stuff in here, especially the idea about hover and eslint. I like that!

However, I think the cursor change is more of a web pattern. You don't see the drag cursor that much on desktop.

**Comment by stephenkingsley** [comment]
@terkelg I agree it! it delete it~

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Ramme <img src="static/icon.png" width="70">
+# Ramme(Stephen.Kingsley process) <img src="static/icon.png" width="70">
```
Comment (terkelg) [inline README.md]: What's going on here?

Comment (stephenkingsley) [inline README.md]: this is my fault! I am interesting in this project, so I want to fork a version to create some awesome! I modify it!

**Comment by terkelg** [comment]
Can you explain your changes to the build script and what the benefits are in 108f9e0?

File: README.md (hunk 1 above)
Comment (terkelg) [inline README.md]: I welcome new ideas and changes! It's the power of open source 👍 
When you create pull-request please create one pull-request pr. feature 😄

**Comment by stephenkingsley** [comment]
@terkelg first, I think the build script is too long to unfriendly. so I create a file to run this.

**Comment by stephenkingsley** [comment]
@terkelg I change back, but I do not delete folde `scripts`, do you want to delete it?

File: README.md (hunk 1 above)
Comment (stephenkingsley) [inline README.md]: sorry this is my fault!

**Comment by stephenkingsley** [comment]
I delete `build` folder

**Comment by terkelg** [comment]
Awesome! I'll look at your pull-request in the weekend and then merge it. Thanks for contributing!

**Comment by terkelg** [comment]
Wow, a lot is going on here! Is it possible that you can split the pull-request pr. feature?
It would make it easier and quicker for me to check and then merge into master. It would also make it easier to reference in the future

**Comment by stephenkingsley** [comment]
can you review code first? this weekend I had other work to do. and I add eslint, so if split pr, there are so much modify in each pr. what do you think?

**Comment by stephenkingsley** [comment]
@terkelg what do you think? I can split it. but every split will also has so much change.

**Pull request closed without merging**